lib/OpenQA/Worker/Commands.pm

```diff
@@ -88,15 +88,16 @@ sub websocket_commands {
     elsif ($msg eq 'livelog_start') {
         # change update_status timer if $job running
         if (backend_running) {
-            $OpenQA::Worker::Jobs::do_livelog = 1;
-            change_timer('update_status', STATUS_UPDATES_FAST);
+            print "starting livelog\n" if $verbose;
+            $OpenQA::Worker::Jobs::do_livelog += 1;
```

Comment:
If I am to trigger it here, I may and will trigger it during already running one. Even then using next_tick() trick. So then I would need some barriers and be rejected anyway and still wait up to 5s.